internal/repository/postgres/employee/employee.go

```diff
@@ -0,0 +1,146 @@
+package employee
```

Comment:
Your PR should contain changes only relevant to the task. E.g. this PR is focusen on Mongo DB integration, so there is no reason to have Postgres in this PR. 

I think the reason for this is that this was branched out from `postgres` branch, instead of `main`. When you start working on a new task you should always branch out from the branch that is actively developed (e.g. `develop` in git flow, `main` or `master` in GitHub flow, etc.)

In some cases (e.g. when you have big chunk of functionality, that depends on each other), such as Epic, you might need to use [stacked diffs](https://newsletter.pragmaticengineer.com/p/stacked-diffs), but in this case you should merge the result not in the `main`, but in the branch you were branching out from. Postgres in this case.

Reply:
i thought, since i am refactoring after adding another `mongo` layer to repo, it is ok to change folders, now i will pay more attention